tensorboard/webapp/widgets/histogram/histogram_component.ng.html

```diff
@@ -50,14 +50,14 @@
       </g>
     </svg>
     <!-- Disable the feature when in non-offset and non-step mode. -->
-    <ng-container *ngIf="isLinkedTimeEnabled(linkedTime)">
+    <ng-container *ngIf="isTimeSelectionEnabled(timeSelection)">
       <histogram-card-fob-controller
         class="linked-time"
-        [linkedTime]="linkedTime"
+        [timeSelection]="timeSelection"
         [steps]="getSteps()"
         [temporalScale]="scales.temporalScale"
-        (onSelectTimeChanged)="onSelectTimeChanged.emit($event)"
-        (onSelectTimeToggle)="onSelectTimeToggle.emit()"
+        (onTimeSelectionChanged)="onLinkedTimeSelectionChanged.emit($event)"
```

Comment:
I do notice an inconsistency here, though. If its purposeful then I'm ok with it.